tests/agent_features/test_attributes_in_action.py

```diff
@@ -165,30 +155,27 @@ def normal_application(request):
 # Tests for checking the presence and format of agent attributes.
 # Test default settings.
 
-_expected_attributes = {"agent": TRACE_ERROR_AGENT_KEYS, "user": ERROR_USER_ATTRS, "intrinsic": ["trip_id"]}
+_expected_attributes = {"agent": TRACE_ERROR_AGENT_KEYS, "user": ERROR_USER_ATTRS, "intrinsic": ["guid"]}
```

Comment:
All of these changes where we deleted trip_id are breaking this file, it's supposed to test for the behavior of intrinsics. I replaced them all with guid and that should solve all those issues.